style.css

```diff
@@ -910,3 +917,1217 @@ footer #contact-form .contact-details p {
   background-color: #fff;
   color: rgb(134, 134, 134);
 }
+
+@media only screen and (min-width: 768px) {
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  body {
+    background-color: #e5e5e5;
+  }
+
+  main {
+    width: fit-content;
+    height: auto;
+  }
+
+  header .mobile-nav {
+    display: none;
+  }
+
+  header .desktop-nav {
+    display: flex;
+    justify-content: space-between;
+    background-color: #fff;
+    position: fixed;
+    width: 100%;
+    height: 72px;
+  }
+
+  .content a {
+    position: absolute;
+    left: 0.8rem;
+    top: 25.5rem;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    width: auto;
+    height: 24px;
+    line-height: 24px;
+    text-decoration: none;
+    color: #7f8cff;
+    margin: auto auto 2rem 9rem;
+  }
+
+  header .desktop-nav a {
+    text-decoration: none;
+  }
+
+  header .desktop-nav h4 {
+    width: 100%;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+    margin-left: 4rem;
+    padding-bottom: 2.8rem;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .icons ul {
+    margin-left: 8rem;
+    margin-top: -0.6rem;
+    display: flex;
+    list-style: none;
+  }
+
+  header .desktop-nav ul {
+    display: flex;
+    list-style: none;
+    margin-right: 4rem;
+  }
+
+  header .desktop-nav li {
+    width: 62px;
+    height: 20px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #344563;
+    justify-content: space-between;
+    margin: 1.5rem;
+  }
+
+  #headline {
+    background-image: url("Images/Header_Desktop.png");
+    background-size: 100%;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
- [ ] Kindly add some margin below the headline section to provide space between the section and the projects section. You are expected to stick to design as much as possible.
### Figma 
![image](https://user-images.githubusercontent.com/94551484/236448929-6f633916-53b5-47c6-826c-91d52db0536e.png)
### your desgin
![image](https://user-images.githubusercontent.com/94551484/236449013-f6f6bb8d-aa71-452b-a7fa-a21e74bd849b.png)


- [ ] I noticed some elements in the headline section overlapping on the desktop version. Kindly fix this please.
![image](https://user-images.githubusercontent.com/94551484/236449171-2b60f0bb-2b67-4e86-b125-555c4fd6800e.png)